### PR TITLE
doc revert reason

### LINKED ISSUE #111: Document revert reason CLI flag
See: https://github.com/ConsenSys/quorum/pull/1215

### PR DESCRIPTION
**Pull request by rolandtyler**
## Pull Request Description
Documented revertreason CLI option and added revertreason concept page

## Fixed Issue(s)

fixes #111

### CODE REVIEW

File: docs/HowTo/Use/Revert-Reason.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+## Enabling revert reason
+
+Use the [`--revertreason`](../../Reference/CLI-Syntax.md#revertreason)
+command line option to include the revert reason in the transaction receipt.
+
+## Where is the revert reason included
+
+With revert reason enabled, the transaction receipt returned by
+[`eth_getTransactionReceipt`](https://eth.wiki/json-rpc/API#eth_getTransactionReceipt) includes
```
Comment (alexandratran) [inline docs/HowTo/Use/Revert-Reason.md:39]:
```suggestion
[`eth_getTransactionReceipt`](https://eth.wiki/json-rpc/API#eth_gettransactionreceipt) includes
```

Sorry we didn't catch this before, but headers in URLs need to be lowercase to work.

File: docs/Reference/CLI-Syntax.md [hunk 2]
```diff
@@ -294,3 +294,20 @@ Defaults to the [`datadir` option](https://geth.ethereum.org/docs/interface/comm
...
+    ```
+
+=== "Command Line"
+
+    ```bash
+    --revertreason
+    ```
+
+Enables including the [revert reason](../HowTo/Use/Revert-Reason.md) in the
+[`eth_getTransactionReceipt`](https://eth.wiki/json-rpc/API#eth_getTransactionReceipt) response.
```
Comment (alexandratran) [inline docs/Reference/CLI-Syntax.md:313]:
```suggestion
[`eth_getTransactionReceipt`](https://eth.wiki/json-rpc/API#eth_gettransactionreceipt) response.
```

Same as previous comment.

**Review by alexandratran — CHANGES_REQUESTED** [review]
Great first issue! Minor change request.

**Review by alexandratran — APPROVED** [review]
LGTM.

**Review by ricardolyn — APPROVED** [review]

**Pull request merged**